File: Hospital.Server/Entities/Response/LabOrderResponse.cs
namespace Hospital.Server.Entities.Response
{
    public class LabOrderResponse
    {
        public long Id { get; set; }
        public long ConsultationId { get; set; }
        public long DoctorId { get; set; }
        public long PatientId { get; set; }
        public string OrderNumber { get; set; } = string.Empty;
        public int OrderStatus { get; set; }
        public decimal TotalAmount { get; set; }
        public bool IsExternal { get; set; }
        public string? Notes { get; set; }
        public int State { get; set; }
        public string CreatedAt { get; set; } = string.Empty;
        public long CreatedBy { get; set; }
        public long? UpdatedBy { get; set; }
        public string? UpdatedAt { get; set; }
    }
}

File: hospital.client/src/types/LabOrderResponse.ts
export interface LabOrderResponse {
  id: number;
  consultationId: number;
  doctorId: number;
  patientId: number;
  orderNumber: string;
  orderStatus: number;
  totalAmount: number;
  isExternal: boolean;
  notes?: string | null;
  state: number;
  createdAt: string;
  createdBy: number;
  updatedBy?: number | null;
  updatedAt?: string | null;
}

export interface LabOrderRequest {
  id?: number | null;
  consultationId?: number | null;
  doctorId?: number | null;
  patientId?: number | null;
  orderNumber?: string | null;
  orderStatus?: number | null;
  totalAmount?: number | null;
  isExternal?: boolean | null;
  notes?: string | null;
  state?: number | null;
  createdBy?: number | null;
  updatedBy?: number | null;
}
